commands/install_license

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+
+    Usage: c8y oee $(basename "$0") [--license <license-string>] [--filePath <path-to-license-file>]
+    Arguments:
+      -l, --license   license-string        The license string
```

Comment:
If I use this option, the license is not valid b/c the double quotes are not properly escaped. Instead they are just removed.